pygpsclient/globals.py

```diff
@@ -136,7 +136,7 @@
     "USB_to_UART",
 )
 # list of available bps rates (first entry in list is the default):
-BPSRATES = (9600, 19200, 38400, 57600, 115200, 230400, 460800, 921600, 4800)
+BPSRATES = (9600, 19200, 38400, 57600, 115200, 230400, 460800, 921600, 4800, 1000000)

```

Comment:
any particular reason for adding the additional non-standard baud rate? Also, can we keep them in ascending order - makes more sense from a user selection standpoint

Reply:
The receiver I'm using to test uses it, and whoops I didn't notice everything before the 4.8k was ascending.

Reply:
> The receiver I'm using to test uses it, and whoops I didn't notice everything before the 4.8k was ascending.

The reason for the order was that I wanted 9600 to be the default, so that's why it's the first in the list. The spinbox then cycles round in order.

Reply:
It should be in the right place now.